tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -42,3 +42,40 @@ dependency,dependencyUrl,licenseOverride
 "org.eclipse.jdt:org.eclipse.jdt.core:3.10.0",z,EPL-1.0
 "org.eclipse.osgi:org.eclipse.osgi:3.7.1",z,EPL-1.0
 "org.eclipse.text:org.eclipse.text:3.5.101",z,EPL-1.0
+
+"addressable:2.3.8",https://github.com/sporkmonger/addressable,Apache-2.0
+"asm:6.0",http://asm.objectweb.org,BSD-3-Clause
+"atomic:1.1.99",http://github.com/ruby-concurrency/atomic,Apache-2.0
+"avl_tree,1.2.1",https://github.com/nahi/avl_tree,BSD-2-Clause-FreeBSD
+"awesome_print:1.8.0",https://github.com/awesome-print/awesome_print,MIT
+"aws-sdk-core:2.11.37",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"aws-sdk-resources:2.11.37",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"aws-sdk-v1:1.67.0",http://aws.amazon.com/sdkforruby,Apache-2.0
+"aws-sdk:2.11.37",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"aws-sigv4:1.0.2",http://github.com/aws/aws-sdk-ruby,Apache-2.0
+"backports:3.11.3",http://github.com/marcandre/backports,MIT
+"bindata:2.4.3",http://github.com/dmendel/bindata,BSD-2-Clause
+"bouncycastle:",https://www.bouncycastle.org/license.html,MIT
+"buftok:0.2.0",https://github.com/sferik/buftok, MIT OR Ruby|BSD-2-Clause
+"builder:3.2.3",http://onestepback.org,MIT
+"bundler:1.9.10",https://bundler.io/,MIT
+"bytelist:",http://github.com/jruby/bytelist,EPL-1.0
+"cabin:0.9.0",https://github.com/jordansissel/ruby-cabin,Apache-2.0
+"childprocess:0.9.0",http://github.com/enkessler/childprocess,MIT
+"chronic_duration:0.10.6",https://github.com/hpoydar/chronic_duration,MIT
+"clamp:0.6.5",http://github.com/mdub/clamp,MIT
+"coderay:1.1.2",http://coderay.rubychan.de,MIT
+"com.fasterxml.jackson.core:jackson-annotations:2.9.5",https://github.com/FasterXML/jackson-annotations,Apache-2.0
+"com.fasterxml.jackson.core:jackson-core:2.9.5",https://github.com/FasterXML/jackson-core,Apache-2.0
+"com.fasterxml.jackson.core:jackson-databind:2.9.5",https://github.com/FasterXML/jackson-databind,Apache-2.0
+"com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.9.5",https://github.com/FasterXML/jackson-dataformats-binary,Apache-2.0
+"com.fasterxml.jackson.module:jackson-module-afterburner:2.9.5",https://github.com/FasterXML/jackson-modules-base,Apache-2.0
+"com.google.googlejavaformat:google-java-format:1.1",https://github.com/google/google-java-format,Apache-2.0
+"com.google.guava:guava:19.0",https://github.com/google/guava,Apache-2.0
+"commons-codec:commons-codec:1.10.0",http://commons.apache.org/proper/commons-codec/,Apache-2.0
+"commons-logging:commons-logging:1.2.0",http://commons.apache.org/proper/commons-logging/,Apache-2.0
+"concurrent-ruby:1.0.5",http://www.concurrent-ruby.com,MIT
+"controls.js:",https://github.com/controlsjs/controls.js,GPL-3.0-only
```

Comment:
No notice file

Reply:
@andrewvc I don't see this being included at all in logstash or jruby (despite being in jruby's copying file), should we keep this?